### ISSUE TITLE
[ACCESSIBILITY]: Add Pittsburgh VAMC content pages back into the daily CI/CD axe checks

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#20120]
## Issue Description
The Pittsburgh VA Medical Center pages are excluded from axe checks that run in the CI/CD content build. These pages should be added to the daily scan.

---
## Tasks
- [ ] Remove the exclusion for Pittsburgh health care pages on [line 17 of the sitemap-helpers.js file](https://github.com/department-of-veterans-affairs/vets-website/blob/aeb0db16ad1d466d1a7c62c3b6bbb19f1e638462/src/platform/site-wide/tests/sitemap/sitemap-helpers.js#L17)
- [ ] Test the content build locally for accessibility errors. This will involve pulling latest content from Drupal, and running the **test:accessibility** command locally.
- [ ] Log any axe violations with the VAMC team

## Acceptance Criteria
- [ ] Change should be announced on relevant Slack channels ahead of PR merge
- [ ] Pittsburgh medical center pages should return 0 axe violations
- [ ] CI/CD should pass all checks on feature branch build

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by pjhill** [comment]
OBE
